src/fread.c

```diff
@@ -2035,7 +2035,10 @@ int freadMain(freadMainArgs _args) {
     }
     if (verbose) {
       if (sampleLines==0) {
-        DTPRINT(_("  'header' determined to be %s because there are%s number fields in the first and only row\n"), args.header?"TRUE":"FALSE", args.header?_(" no"):"");
+        if (args.header)
```

Comment:
Looks odd to me to branch on `args.header` and then template on `args.header ? T : F`. Let's just use `TRUE` / `FALSE` directly.